nestor_api/lib/io.py

```diff
@@ -86,12 +86,23 @@ def get_temporary_directory_path(prefix=""):
     return get_working_path(tmp_directory_name)
 
 
-def get_working_path(working_path_name):
+def get_working_path(working_path_name: str) -> str:
     """Returns the working path"""
     return os.path.join(Configuration.get_working_path(), working_path_name)
 
 
-def remove(file_path):
+def to_yaml(data: dict) -> str:
+    """Converts a dictionary into a valid yaml string"""
+    return yaml.safe_dump(data)
+
+
+def read(file_path: str) -> str:
+    """Read the file content at the given path"""
+    file = open(file_path, "r")
+    return file.read()
```

Comment:
You forgot to call `file.close()`.
I recommend to use the `with` structure to ensure this
```python
def read(file_path: str) -> str:
    """Read the file content at the given path"""
    with open(file_path, "r") as file:
        file_content = file.read()
    return file_content
```